tests/unit/core/test_docs_util.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding:utf-8 -*-
+#
+# Copyright 2019 Victor Torre
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import testtools
+
+from bandit.core.docs_utils import BASE_URL
+from bandit.core.docs_utils import get_url
+
+
+class UtilTests(testtools.TestCase):
```

Comment:
There is already a UtilTests defined in test_util.py. I'd rather avoid duplicate class name here.  DocsUtilTests makes more sense.